bazel-haskell-deps.bzl

```diff
@@ -17,8 +17,8 @@ load("@os_info//:os_info.bzl", "is_linux", "is_windows")
 load("@dadew//:dadew.bzl", "dadew_tool_home")
 load("@rules_haskell//haskell:cabal.bzl", "stack_snapshot")
 
-GHCIDE_REV = "2ba61893ea07df26dcacf165a79dd6d0381a3dc8"
-GHCIDE_SHA256 = "f3a2fb693d1e6d3beda542639de027a94ae3417fe483765510eceb2ad9ea2b29"
+GHCIDE_REV = "19475c57fca859a135b5304784a3cb027cd51407"
```

Comment:
This is the [corresponding `daml-ghcide` PR](https://github.com/digital-asset/daml-ghcide/pull/18)